README.md

```diff
@@ -1,15 +1,48 @@
-#### Odoo Project Tasks
+# Odoo Project Tools
 
+This repository contains helper tasks for working with Camptocamp Odoo projects.
 
-#### Installation
+## Installation
 
-directly to os
+Create a virtual environment and activate it, then
 
 ```
-pip install --user git+ssh://git@github.com/yankinmax/odoo-project-tasks@BSRD-629#egg=invoke_cli
+pip install git+ssh://git@github.com/camptocamp/odoo-project-tools#egg=odoo-tools
 ```
 
-to virtualenv
-```
-pip install git+ssh://git@github.com/yankinmax/odoo-project-tasks@BSRD-629#egg=invoke_cli
-```
+
+## Usage
+
+Note: information below is subject to change.
+
+
+The package brings the following commands:
+
+
+`otools-project init`: initialize a new project
+
+`otools-addons`: tools to work with addons
+
+Commands:
+
+Usage: otools-addon add [OPTIONS] NAME
```

Comment:
IMO we should not document everything here. 
We should take care more of the automatic documentation for each tool.
Here I would name only the main commands and ask to run the commands for more details.
There we can also provide examples.

Reply:
We can document otools --help ^^

I agree with @simahawk we want the tool to be self explanatory.